frontends/web/src/api/subscribe2.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2021 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { subscribe, IUnsubscribe } from '../utils/event2';
```

Comment:
What's the deal with event2 and subscribe2? I'm not sure why there are two of them > confusing > please rename.

Reply:
Yes, but I am not sure how to name them, maybe eventOld.ts and subscribeOld.ts ?

The difference is old is sending a message with Type, Data and maybe meta, deviceID or code.

Vs. the new message that send subject, action and object and afaik subject is also available via GET api. i.e. `Subject: "devices/registered"` could also be fetched by the frontend via api `devices/registered`

See we have old-school and new-school https://github.com/digitalbitbox/bitbox-wallet-app/blob/569537fa3ec01a6247d92e9b71cb422c325c952f/backend/backend.go#L1067-L1079

Reply:
renamed to event-legacy.ts and subscribe-legacy.ts